airbyte-integrations/bases/base-normalization/normalization/transform_catalog/stream_processor.py

```diff
@@ -487,29 +487,34 @@ def generate_dedup_record_model(self, from_table: str, column_names: Dict[str, T
     def generate_scd_type_2_model(self, from_table: str, column_names: Dict[str, Tuple[str, str]]) -> str:
         template = Template(
             """
--- SQL model to build a Type 2 Slowly Changing Dimension (SCD) table for each record identified by their primary key
-select
-  {%- if parent_hash_id %}
-    {{ parent_hash_id }},
-  {%- endif %}
-  {%- for field in fields %}
-    {{ field }},
-  {%- endfor %}
-    {{ cursor_field }} as _airbyte_start_at,
-    lag({{ cursor_field }}) over (
-        partition by {{ primary_key }}
-        order by {{ cursor_field }} desc, _airbyte_emitted_at desc
-    ) as _airbyte_end_at,
-    lag({{ cursor_field }}) over (
-        partition by {{ primary_key }}
-        order by {{ cursor_field }} desc, _airbyte_emitted_at desc
-    ) is null as _airbyte_active_row,
-    _airbyte_emitted_at,
-    {{ hash_id }}
-from {{ from_table }}
-{{ sql_table_comment }}
+        -- SQL model to build a Type 2 Slowly Changing Dimension (SCD) table for each record identified by their primary key
```

Comment:
Not sure if we should be re-indenting here... Is this made to re-align with the python code?
(the generated files will be indented following this template too)